### PR TITLE
[ADD] a more load button in POS debt history

### PR DESCRIPTION
**Pull request by ufaks**

### CODE REVIEW

File: pos_debt_notebook/static/src/js/pos.js [hunk 1]
```diff
@@ -450,6 +452,25 @@ odoo.define('pos_debt_notebook.pos', function (require) {
                     debt_history_line = debt_history_line.childNodes[1];
                     contents.appendChild(debt_history_line);
                 }
+                if (debt_history.length % this.debt_history_limit === 0) {
+                    var debt_history_load_more_html = QWeb.render('DebtHistoryLoadMore');
+                    var debt_history_load_more = document.createElement('tbody');
+                    debt_history_load_more.innerHTML = debt_history_load_more_html;
+                    debt_history_load_more = debt_history_load_more.childNodes[1];
+                    contents.appendChild(debt_history_load_more);
+                    var new_limit = $('#debt_history tr').length - 2 + this.debt_history_limit;
```
Comment (yelizariev) [inline pos_debt_notebook/static/src/js/pos.js]: You cannot do it according to mvc approach. Fix that please

**Review by yelizariev — CHANGES_REQUESTED** [review]
https://en.m.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller

File: pos_debt_notebook/static/src/js/pos.js [hunk 2]
```diff
@@ -398,6 +405,8 @@ odoo.define('pos_debt_notebook.pos', function (require) {
             this.pos.on('updateDebtHistory', function(partner_ids){
                 this.update_debt_history(partner_ids);
             }, this);
+            this.debt_history_limit = 10;
+            this.debt_history_built_lines = 0;
```
Comment (yelizariev) [inline pos_debt_notebook/static/src/js/pos.js]: Can it be more clear names? Also, use a separate setting for initial limit.

E.g.

            this.debt_history_limit_initial = 10;
            this.debt_history_limit_increment = 20;
            this.debt_history_built_current = 0;

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by manu686** [comment]
yay 👍

**Pull request merged**